.github/workflows/links.yml

```diff
@@ -0,0 +1,20 @@
+name: Links
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 18 * * *"
```

Comment:
@lakshya8066 I see the issue now, it's not running because There hasn't been a repository_dispatch or a workflow_dispatch event. And the cron is waiting on its schedule.

If we want it to happen during pull_requests, add `pull_request:` and I would even add `push:`

Reply:
Oh yeah, that might be the issue...